webanno-api-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/api/annotation/rendering/model/VSpan.java

```diff
@@ -41,7 +41,7 @@ public VSpan(AnnotationLayer aLayer, AnnotationFS aFS, String aType, VRange aOff
             Map<String, String> aFeatures, Map<String, String> aHoverFeatures)
     {
         this(aLayer, new VID(getAddr(aFS)), aType, asList(aOffsets), aFeatures, 
-                aHoverFeatures, null);
+                aHoverFeatures, null, null);
```

Comment:
I'd suggest to not add the action buttons flag to the constructor but rather set it always via a setter. The constructor signature is already large enough.